lib/systems/default.nix

```diff
@@ -112,15 +131,25 @@ let
         else if final.isNone                  then "newlib"
         # TODO(@Ericson2314) think more about other operating systems
         else                                     "native/impure";
+
+      unwinderlib =
+        /**/ if final.useLLVM || final.useArocc || final.useZig then "libunwind"
+        else if final.isDarwin then "libunwind-system"
+        else "libgcc_s";
```

Comment:
`libgcc_s` is just the shared library form of `libgcc`; maybe this should just be `libgcc`? (Can you mix and match `unwinderlib` and `rtlib`, or are the possible cases actually just `libgcc`, `compiler-rt` + `libunwind`, and `compiler-rt` + system `libunwind`? If the latter, then perhaps we shouldn’t introduce another variable for the unwinder here.)